File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/EldritchMitre.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory;

/* JADX INFO: loaded from: classes3.dex */
public class EldritchMitre extends Accessory {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.accessory_eldritch_mitre_name;
        this.idDescription = R.string.accessory_eldritch_mitre_description;
        this.idEffect = R.string.accessory_eldritch_mitre_effect;
        this.idImage = R.drawable.eldritch_mitre;
        this.price = 2255L;
        this.maxHp = 50;
        this.intelligence = 45;
        this.healingModifier = -0.99d;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/EldritchMitre.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.abstractClasses.Accessory

class EldritchMitre : Accessory() {
    override fun configureProperties() {
        idName = R.string.accessory_eldritch_mitre_name
        idDescription = R.string.accessory_eldritch_mitre_description
        idEffect = R.string.accessory_eldritch_mitre_effect
        idImage = R.drawable.eldritch_mitre
        price = 2255L
        maxHp = 50
        intelligence = 45
        healingModifier = -0.99
    }
}
